requirments/REQUIREMENTS.md

```diff
@@ -0,0 +1,35 @@
+## Roles
+#### Unauthorized user
+Unathorized users can view books and comments to them, but cannot purchase books or subscriptions. Also they can sign in.
+#### Authorized user
+Authorized users can view books, purchase books and subscriptions, view and leave comments. Also they can sign out.
+#### Moderator
```

Comment:
Are Moderators, Copyright holders and Service Managers signed on somehow? Looks like the authentication part of all those roles is not represented at all :)